src/psyclone/tests/domain/lfric/lfric_scalar_mdata_test.py

```diff
@@ -35,6 +35,7 @@
 #         I. Kavcic and A. Coughtrie, Met Office;
 #         C. M. Maynard, Met Office/University of Reading;
 #         J. Henrichs, Bureau of Meteorology.
+# Modified by L. Turner, Met Office
```

Comment:
I would suggest moving your name with all authors in MO line 35, since this seems to be the convention for this module.